sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/pf_templates/built_in_metrics/chat/flow.dag.yaml

```diff
@@ -132,7 +132,7 @@ nodes:
     type: code
     path: evaluate_chat_rag.py
   inputs:
-    connection: Default_AzureOpenAI
+    connection: openai_connection
```

Comment:
Is this change intended?

Reply:
Yes it is. This is due to some limitation on prompt flow that we have to do this. Happy to discuss it in more details if needed.